src/Integrations/Mailchimp/Mailchimp.php

```diff
@@ -60,7 +60,7 @@ class Mailchimp extends AbstractFormBuilder implements MapperInterface, ServiceI
 	 *
 	 * @var ValidatorInterface
 	 */
-	public $validator;
+	protected $validator;
```

Comment:
Same as the above. Whenever you are injecting something in the class, and you don't need the property to be in a child class, but used only in the class where it's injected, you need to use `private` visibility.

This helps with the encapsulation of the object's state.

If you want to use the property you'd still wouldn't want to set it to `public`, but would instead set a specific getter for it.